packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -83,7 +83,7 @@ contract FederatedAttestations is
   /**
    * @notice Sets the EIP712 domain separator for the Celo FederatedAttestations abstraction.
    */
-  function setEip712DomainSeparator() public {
+  function setEip712DomainSeparator() external {
```

Comment:
Not sure why this function is `public` in other contracts (ex. Accounts, MetaTransactionWallet)